services/GraphQL/queries/users.ts

```diff
@@ -0,0 +1,41 @@
+import { gql } from '@apollo/client'
+
+export const GET_USER_SESSION = gql`
+  query login($password: String = "", $username: String = "") {
+    users(where: { password: { _eq: $password }, username: { _eq: $username } }) {
+      username
+      type
+      is_active
+      id
+      created_at
+      avatar_url
+    }
+  }
+`
+
+export const GET_USER_BY_ID = gql`
+  query ($id: String!) {
+    user(id: $id) {
+      username
+      type
+      is_active
+      id
+      created_at
+      avatar_url
+    }
+  }
+`
```

Comment:
I didn't know this way to filter by ID, is there a difference between user(id: $id) and `where`?

Reply:
I verified and updated, that way was not working